src/Microsoft.AspNetCore.Testing/HttpClientSlim.cs

```diff
@@ -78,6 +78,12 @@ private static HttpStatusCode GetStatus(string response)
             var statusStart = response.IndexOf(' ') + 1;
             var statusEnd = response.IndexOf(' ', statusStart) - 1;
             var statusLength = statusEnd - statusStart + 1;
+
+            if (statusLength < 1)
+            {
+                throw new FormatException($"No StatusCode found in '{response}'");
```

Comment:
I'm not convinced `FormatException` is right here, so open to suggestions.

Reply:
I'd go with InvalidDataException or InvalidOperationException